src/samples/UWP/Uno.Themes.Samples.Shared/Entities/Data/TestCommands.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Windows.Input;
+using static Uno.Themes.Samples.ViewModels.ViewModelBase;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.Themes.Samples.Entities.Data
+{
+    public class TestCommands
+    {
+		public ICommand MyCommand { get; private set; }
```

Comment:
rename MyCommand to ShowWarningCommand
OR
rename MyCommand to ShowWarning && rename ShowWarning to ShowWarningImpl